src/js/API/peaks/methods.ts

```diff
@@ -83,3 +105,52 @@ export const getNumberOfPeakConquerors = async (
     return undefined;
   }
 };
+
+export const getPeakAverageTimeCompletion = async (
+  peakId: string
+): Promise<number | undefined> => {
+  try {
+    const response = await axios.get<number>(
+      getPeakAverageTimeCompletionUrl(peakId),
+      {
+        headers: authHeader(),
+      }
+    );
+    return response.data;
+  } catch (error) {
+    console.log(error);
+    return undefined;
+  }
+};
+
+export const getPeakPosts = async (peakId: string): Promise<Post[]> => {
+  try {
+    const response = await axios.get(getPeakPostsUrl(peakId), {
+      headers: authHeader(),
+    });
+    return response.data.content;
+  } catch (error) {
+    console.log(error);
+    return [];
+  }
+};
+
+export const getPeakPostsPaginated = (
+  peakId: string,
+  pageSize: number
+) => async (page: number): Promise<{ pages: number; data: Post[] }> => {
+  try {
+    const pagePeakPostsUrl =
+      getPeakPostsUrl(peakId) + `?pageNumber=${page}&pageSize=${pageSize}`;
```

Comment:
Kwestia konwencji, ale tworzenie takiego dynamicznego url z parametrami przeniósłbym do funkcji

Reply:
To ja też to podmienię.